api/schemas.py

```diff
@@ -12,3 +12,17 @@ class PayloadSchema(Schema):
     date = fields.Date(required=True, validate=validate_date)
     hd = fields.Boolean(required=True)
     thumbs = fields.Boolean(missing=True)
+
+
+class CMEPayloadSchema(Schema):
+    startDate = fields.Date(format='%Y-%m-%d',
+                            required=True)
+    endDate = fields.Date(format='%Y-%m-%d',
+                          required=True)
+
+    @validates_schema()
+    def validate_date(self, data, **kwargs):
```

Comment:
I find it a bit confusing that this file has `validate_date` names repetitive.

Reply:
https://github.com/mstoro/nasa_api/pull/3/commits/ab6cdcf191a46490a7c5e6356d9905d10d2be676